$OpenBSD: patch-configure,v 1.11 2020/02/10 18:06:34 jasper Exp $

- Link with libc++abi on arm (to access unwind symbols)

Index: configure
--- configure.orig
+++ configure
@@ -18475,6 +18475,12 @@ $as_echo "#define SYMBOL_UNDERSCORE 1" >>confdefs.h
 
 fi
 
+case "$target" in
+    arm-*-openbsd*)
+      LDFLAGS="-lc++abi $LDFLAGS"
+    ;;
+esac
+
 FFI_EXEC_TRAMPOLINE_TABLE=0
 case "$target" in
      *arm*-apple-* | aarch64-apple-*)
